src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/addressSearch/OsPlacesApiClient.kt

```diff
@@ -34,7 +32,7 @@ class OsPlacesApiClient(
       .accept(MediaType.APPLICATION_JSON)
       .retrieve()
       .bodyToMono(OsPlacesApiResponse::class.java)
-      .onErrorResume { e -> if (e is WebClientResponseException && e.statusCode == HttpStatus.BAD_REQUEST) Mono.empty() else Mono.error(e) }
```

Comment:
not entirely sure why we are doing this copied from afer,.. assume it's to prevent errors occurring when user types in something funny! Is this really going to happen?

Reply:
we also do this on post code 
`val cleanedPostcode = postcode.uppercase().replace(Regex("[^A-Z0-9]"), "")`

Reply:
Ideally frontend validation should prevent this from being triggered, so we can treat this as an unexpected error so don't need specific error handling (assuming default error handling will provide enough info).

Reply:
Thats what I was wondering....